policy-engine/src/graph.rs

```diff
@@ -144,9 +147,30 @@ where
     let graph_json = serde_json::to_string(&internal_io.graph)
         .map_err(|e| GraphError::FailedJsonOut(e.to_string()))?;
 
+    let v =
+        serde_json::from_str(&graph_json).map_err(|e| GraphError::FailedJsonOut(e.to_string()))?;
+    let version: i32 = match internal_io.parameters.get("version") {
+        None => *commons::MIN_CINCI_VERSION as i32,
+        Some(v) => v.parse::<i32>().unwrap(),
+    };
+
+    let updated_json = add_version_information(&v, &version);
```

Comment:
This feels like it's cheating a bit and adding a hard-coded plugin to the tail of the plugin chain.  On the one hand, we have a number of hard-coded policy-engine configs in places like cincinnati-operator where this is deployed.  On the other, we can bump all those locations to configure a new version endpoint.  I'd rather make this an official plugin, since it's consuming information from `plugin_params`.